src/fromYaml.js

```diff
@@ -58,13 +82,19 @@ const fromYaml = (content) => {
     }
 
     const { order, title, name, url, pages } = object
+    const convertedPages = convertPages(pages)
 
+    let homePage = url
+    if (!homePage || homePage === undefined || homePage === '/') {
```

Comment:
Why `homePage === '/'`?

Reply:
Saw in this test, https://github.com/adobe/parliament-transformer-navigation/blob/main/src/__tests__/fromYaml.test.js#L21 that root url might be explicitly defined as '/'.